### ISSUE TITLE
[Link]:  Screen reader announcement does not match the visually provided link label

### ISSUE THREAD

**Issue by AlexYanSap1** [UI5/webcomponents#13276]
### Bug Description

The name announced by screen readers for <ui5-link> does not match the visible link text.

Example:

Visible label: Linked Goals (3)

Screen reader announcement: View 3 linked goals

This creates an inconsistency between what sighted users see and what screen reader users perceive.

### Affected Component

Link

### Steps to Reproduce

1. in the Link doc https://ui5.github.io/webcomponents/components/Link/ 
2. edit the sample and add the <ui5-link href="https://www.sap.com" target="_blank" accessible-name="View 3 linked goals">Linked Goals (3)</ui5-link>
3. use the screen reader to focus on the link and see the announcement of the screen reader only includes the accessible name "View 3 linked goals"

<img width="1036" height="357" alt="Image" src="https://github.com/user-attachments/assets/bf0b5373-671b-4d4e-bb40-d2e29c805fe4" />

### Priority

None

### UI5 Web Components Version

2.20.0

### Browser

Chrome

### Additional Context

Although I think it's valid to only announce the a11y name, the test team believe it violates the a11y criteria.

### Organization

SAP SF

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by hinzzx** [comment]
Hi @UI5/ui5-team-ctr-bal ,

The behaviour that the reporter explained is reproducible, and could be checked in this [sample](https://ui5.github.io/webcomponents/play/#eNp9UtuK2zAQ_RWtnpIQWS1LYXGdUrq9vBRa6LZQMBRZmsTayJKRlLgh5N87ku2SpRdssOfM7cyZOVNtFfws2tgZWp6pFR3Q8hpcU-lsBBsRrm4YI70Rp513B6vY1hlFGHtV2-rm7af7h--f35GUlID0JUbY3aamYGuKWHqqFoTCf4JvFeLJQDJWpXVxUSrYagtqSc6IKR1Sq5JYZ-ElApeUn-tn_9Z5CYpJZ5xnQj0eQnwSSyo-109GB1EQ2QofICKlrw_v2V1mNbnS5IgfNQy987GmZJobwUGr2G4UHLUElo010VZHLQwLUhjYPC-eTcWijtjzi-h6AxUfLRybT3Pjb-PUiWRqWLoRcj-pmScpyVH4BWNB9G9-u-6TZzk1-MsOGNhpDyngoF8wo-2etB622KKNsQ8l58MwFFi2kK7D6aLwu6zEjwa3tEdkppQHLMmdhxQnpIQQdGOATRJ9Q4nILUktQJGdEyaM1D6OyIeEkMXtMu9gZvMv7iNvvAXpdR9JPPWpR-fUwUAi5WUyhbbFY2qDS82BWdOk5Kgpn8_uv-rQy3oudXXsM3J96Xx1XaXVCsiK11Z36TZITQt-5Q6HPsGZIB7fn7mZwZP81ygLH6DBXfR4sjYGjvceeZJwrEMvl18H7j5A).

Could you please check ?

Best Regards,
Stoyan

**Comment by GDamyanov** [comment]
Dear @AlexYanSap1,

it is standard screen reader behavior to read the aria-label instead of the text content it is reproducible with native html: https://jsfiddle.net/3pbw2u78/1/ . About the accessible-name attribute it is recommended to be used for links which contains only icons.

Best Regards,
Georgi
